tests/test_output_display.py

```diff
@@ -24,3 +24,8 @@ def test_rad(self):
         self.assertAlmostEqual(display.rad(290.57), math.radians(290.57), 9)
         self.assertAlmostEqual(display.rad(math.pi), math.radians(math.pi), 9)
         self.assertAlmostEqual(display.rad(-11), math.radians(-11), 9)
+
+    def test_binary(self):  # 0b is removed from bin result for all unit tests
+        self.assertEqual((display.convert_to_binary(6)), bin(6).replace("0b", ""))  # Test for positive integer
+        self.assertEqual((display.convert_to_binary(-6)), bin(-6).replace("0b", ""))  # Test for negative integer
```

Comment:
You could always hard code the reference values if there is no decimal binary function available in Python.

Reply:
Oh I wasn't sure if that would still be considered unit testing. Thanks, I'll do this for the next iteration.

Reply:
I think that it is as long as the reference value is correct but maybe the others disagree. We might not have a choice though. Alternatively, maybe there is a library that does what you need?